### PR TITLE
BZ1760123 - update Restore snapshot module

### PR DESCRIPTION
**Pull request by bobfuru**
Reopened [1760123](https://bugzilla.redhat.com/show_bug.cgi?id=1760123) to combine work described [in this comment](https://bugzilla.redhat.com/show_bug.cgi?id=1760123#c3) with existing steps for Restore Snapshot module. Instead of existing as additional steps, the content should be integrated into steps 1 and 2.

### CODE REVIEW

**Comment by bobfuru** [comment]
@dciancio @liangxia PTAL

**Comment by liangxia** [comment]
LGTM

File: modules/persistent-storage-snapshots-restore.adoc [hunk 1]
```diff
@@ -30,42 +34,14 @@ metadata:
     snapshot.alpha.kubernetes.io/snapshot: snapshot-demo <1>
 spec:
   storageClassName: snapshot-promoter <2>
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
```
Comment (dciancio) [inline modules/persistent-storage-snapshots-restore.adoc]: The storage attribute specified is an example size of 1Gi but there should be a note stating to set the storage size to at least accommodate the original PV size.

**Review by dciancio — CHANGES_REQUESTED** [review]
See embedded comment.

File: modules/persistent-storage-snapshots-restore.adoc (hunk 1 above)
Comment (bobfuru) [reply modules/persistent-storage-snapshots-restore.adoc]: Added annotation: "Storage size for a restored snapshot must be large enough to accommodate the original PV size."

**Comment by bobfuru** [comment]
@dciancio Can I please get an ack that these changes look good?

**Review by dciancio — APPROVED** [review]
@bobfuru  - LGTM - Approved.

**Comment by bobfuru** [comment]
@openshift/team-documentation PTAL

**Comment by sheriff-rh** [comment]
Peer review done - looks great, Bob!

**Pull request merged**